examples/datasets/plot_nutrimouse.py

```diff
@@ -0,0 +1,171 @@
+"""
+=====================================================
+An mvlearn casestudy: the Nutrimouse dataset
+=====================================================
+
+In this tutorial, we show how one may utilize various tools of mvlearn. We
+demonstrate applications to the Nutrimouse dataset from a nutrition study on
+mice. The data measures 40 mice and has two views: expression levels of
+potentially relevant genes and concentrations of certain fatty acids. Each
+mouse has two labels: it's genetic type and diet.
+
+[1] P. Martin, H. Guillou, F. Lasserre, S. Déjean, A. Lan, J-M.
+    Pascussi, M. San Cristobal, P. Legrand, P. Besse, T. Pineau.
+    "Novel aspects of PPARalpha-mediated regulation of lipid and
+    xenobiotic metabolism revealed through a nutrigenomic study."
+    Hepatology, 2007.
+"""
+
+# Authors: Ronan Perry
+#
+# License: MIT
+
+###############################################################################
+# Load the Nutrimouse dataset
+# ---------------------------
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import cm
+import seaborn as sns
+from mvlearn.datasets import load_nutrimouse
+
+dataset = load_nutrimouse()
+Xs = [dataset['gene'], dataset['lipid']]
+y = np.vstack((dataset['genotype'], dataset['diet'])).T
+
+print(f"Shapes of each view: {[X.shape for X in Xs]}")
+
+###############################################################################
+# Embed using MVMDS
+# -----------------
+#
+# Multiview multi-dimensional scaling embeds multiview data into a single
+# representation that captures information shared between both views.
+# Embedding the two nutrimouse views, we can observe clear separation between
+# the different genotypes and some of the diets too.
+
+from mvlearn.embed import MVMDS  # noqa: E402
+
+X_mvmds = MVMDS(n_components=2, num_iter=50).fit_transform(Xs)
+
+diet_names = dataset['diet_names']
+genotype_names = dataset['genotype_names']
+plt.figure(figsize=(5, 5))
+for genotype_idx, cmap in enumerate((cm.Blues, cm.Reds)):
+    for diet_idx in range(5):
+        X_idx = np.where((y == (genotype_idx, diet_idx)).all(axis=1))
+        color = cmap((diet_idx + 1) / 6)
+        label = diet_names[diet_idx] + f' ({genotype_names[genotype_idx]})'
+        plt.scatter(*zip(*X_mvmds[X_idx]), color=color, label=label)
```

Comment:
you should use both color and a marker. Also you use a continuous color scale for a categorical (unordered information). Can you try to improve?